clearnode/custody.go

```diff
@@ -381,14 +384,16 @@ func (c *Custody) handleChallenged(logger Logger, ev *nitrolite.CustodyChallenge
 			return fmt.Errorf("error saving channel in database: %w", err)
 		}
 
+		logger.Info("challenged channel", "channelId", channelID)
```

Comment:
```suggestion
		logger.Info("handled challenged event", "channelId", channelID)
```

This function does not challenge channel, it just stores `Challenged` event to the DB.